auxiliary/LidoWstEthToStEthRatioOracle/test/integration/LidoWstEthToStEthRatioOracle.test.ts

```diff
@@ -1,25 +1,24 @@
 import assertBn from '@synthetixio/core-utils/utils/assertions/assert-bignumber';
 import { wei } from '@synthetixio/wei';
-import { bootstrap } from '../../bootstrap';
-import { bn, genBootstrap, genNumber } from '../../generators';
+import { bootstrap } from '../bootstrap';
 
-describe('ChainlinkWstEthNode', () => {
-  const bs = bootstrap(genBootstrap());
-  const { systems, extras, restore } = bs;
+const bn = (n: number) => wei(n).toBN();
```

Comment:
bootstrap file is exporting a `bn` helper already

Reply:
Removed from bootstrap file.